Bluetooth: Mesh: Light Ctrl sample doc fixes (#2991)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/light_ctrl/README.rst

The models are used for the following purposes:

- The first element contains a Config Server and a Health Server.
  The Config Server allows configurator devices to configure the node remotely.
  The Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
  These callbacks trigger blinking of the LEDs.
- The seven other models in the first element are the product of a single instance of the Light Lightness Server.
  The application implements callbacks for the Light Lightness Server to control the first LED on the device using the PWM driver.
- The three models in the second element are the product of a single instance of the Light Control Server.
  The Light Control Server controls the Light Lightness Server in the first element, deciding parameters such as fade time, lighting levels for different states, and inactivity timing.

>   In this sample, the Light Control Server is enabled by default on boot up.

Reviewer: ```suggestion
  In this sample, the Light Control Server is enabled by default at startup.
```

Building and running
********************

> .. |sample path| replace:: :file:`samples/bluetooth/mesh/light_ctrl`

Reviewer 2: Btw, the `sample.yaml` also needs to be updated with a similar change.

Author: Thanks!
